### PR TITLE
feat: add bet365 default theme - WIP

### PR DESCRIPTION
**Pull request by MrTibbles**
## Changes
- Operator: bet365
- Theme: default
- Sport: Golf
- Adds WIP default theme for bet365

## Notes
This is to enable the development of bet365 UI variants in the GEC, changes are to be expected.

### CODE REVIEW

File: stylesheets/golf/bet365-default.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  --baseJetBlack: var(--iconJetBlack);
+}
+
+#groupsSearchClose {
+  --baseJetBlack: var(--iconJetBlack);
+}
+
+#groupsSelectedRound {
+  --baseJetBlack: var(--iconJetBlack);
+}
```
Comment (MrTibbles) [inline stylesheets/golf/bet365-default.css:157]: shouldn't all jet blacks have the same value? or am i being pedantic?

Comment (footerz) [reply stylesheets/golf/bet365-default.css:158]: So, baseJetBlack has been changed to #383838 (dark colour for backgrounds etc) for Bet365. On these components, it needs to change to #dddddd (light colour for text purposes).

iconJetBlack has been changed to #dddddd for Bet365.

I hope that helps, dont know if I understand your question fully.

Comment (footerz) [reply stylesheets/golf/bet365-default.css:158]: I'll be raising Groups PR after leaderboard goes in, so you can see where these are applied, if that helps a bit more...

**Comment by footerz** [comment]
@MrTibbles should we look to get this in now as a first pass.

**Pull request merged**